testing/test_metal_context.mm

```diff
@@ -71,36 +80,37 @@
 
 TestMetalContext::TextureInfo TestMetalContext::CreateMetalTexture(const SkISize& size) {
   std::scoped_lock lock(textures_mutex_);
-  auto texture_descriptor = fml::scoped_nsobject{
-      [[MTLTextureDescriptor texture2DDescriptorWithPixelFormat:MTLPixelFormatBGRA8Unorm
-                                                          width:size.width()
-                                                         height:size.height()
-                                                      mipmapped:NO] retain]};
+  MTLTextureDescriptor* texture_descriptor =
+      [MTLTextureDescriptor texture2DDescriptorWithPixelFormat:MTLPixelFormatBGRA8Unorm
+                                                         width:size.width()
+                                                        height:size.height()
+                                                     mipmapped:NO];
 
   // The most pessimistic option and disables all optimizations but allows tests
   // the most flexible access to the surface. They may read and write to the
   // surface from shaders or use as a pixel view.
-  texture_descriptor.get().usage = MTLTextureUsageUnknown;
+  texture_descriptor.usage = MTLTextureUsageUnknown;
 
   if (!texture_descriptor) {
     FML_CHECK(false) << "Invalid texture descriptor.";
     return {.texture_id = -1, .texture = nullptr};
   }
 
   id<MTLDevice> device = (__bridge id<MTLDevice>)GetMetalDevice();
-  sk_cfp<void*> texture = sk_cfp<void*>{[device newTextureWithDescriptor:texture_descriptor.get()]};
-
+  id<MTLTexture> texture = [device newTextureWithDescriptor:texture_descriptor];
   if (!texture) {
     FML_CHECK(false) << "Could not create texture from texture descriptor.";
     return {.texture_id = -1, .texture = nullptr};
   }
 
   const int64_t texture_id = texture_id_ctr_++;
-  textures_[texture_id] = texture;
+  sk_cfp<void*> texture_ptr;
+  texture_ptr.reset((__bridge_retained void*)texture);
```

Comment:
`retain` + `__bridge` like the other PR?

Reply:
Oh crazy -- nice catch. I thought I'd hit all these, but that might have been in a different local branch. Thanks for spotting.